<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SingleLoad</class>
 <widget class="QWidget" name="SingleLoad">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>927</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Messung laden</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/single_load/res/single_load/icon.png</normaloff>:/single_load/res/single_load/icon.png</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="layout_left">
     <item>
      <widget class="QTreeView" name="dicom_data_tree">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="layout_right">
     <item>
      <widget class="QmitkStdMultiWidget" name="mitkView" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_start">
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">padding: 20px;</string>
       </property>
       <property name="text">
        <string>Diagnosis durchf√ºhren!</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QmitkStdMultiWidget</class>
   <extends>QWidget</extends>
   <header>QmitkStdMultiWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
